### ISSUE TITLE
@simon-rob  Hi, I have added the diou-loss, but the ciouloss will result in Nan loss. According to my result, DIOU got 79.0 for mAP50, maybe we should tune the hyperparameter?  You may try it yourself.

### ISSUE THREAD

**Issue by wlguan** [wlguan/Stronger-yolo-pytorch#20]
@simon-rob  Hi, I have added the diou-loss, but the ciouloss will result in Nan loss. According to my result, DIOU got 79.0 for mAP50, maybe we should tune the hyperparameter?  You may try it yourself.

_Originally posted by @wlguan in https://github.com/wlguan/Stronger-yolo-pytorch/issues/10#issuecomment-559778704_
